### PR TITLE
904698-prepared UG document for volume 3 moved features

### PR DESCRIPTION
**Pull request by SivaranjithN**
Hi Team,

I have prepared the UG document for the below Volume 3 moved features for .NET MAUI editors controls.

- PercentDisplayMode
- MaximumNumberDecimalDigits
- ReturnType

Regards,
Sivaranjith N.

### CODE REVIEW

File: MAUI/Autocomplete/UI-Customization.md [hunk 1]
```diff
@@ -1212,3 +1212,31 @@ N> Dynamic changes to the `HorizontalTextAlignment` property may not be function
 The following image illustrates the result of the above code:
 
 ![TextAlignment](Images/UICustomization/TextAlignment.png)
+
+## ReturnType
+
+The `ReturnType` property specifies the action button (e.g., Next, Done, Go) on the virtual keyboard. It helps manage the flow between multiple input fields by defining what happens when the action button is pressed.
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/UI-Customization.md:1218]: remove virtual keyboard.

File: MAUI/Autocomplete/UI-Customization.md [hunk 2]
```diff
@@ -1212,3 +1212,31 @@ N> Dynamic changes to the `HorizontalTextAlignment` property may not be function
...
+{% endhighlight %}
+{% highlight C# %}
+
+SfAutocomplete sfAutocomplete = new SfAutocomplete();
+sfAutocomplete.ReturnType = ReturnType.Next;
+
+{% endhighlight %}
+{% endtabs %}
+
+![RetrunType](Images/UICustomization/ReturnType.png)
```
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/UI-Customization.md:1242]: check the spelling mistake

File: MAUI/Masked-Entry/MaskedEntry_Images/MaskedEntry_ReturnType.png
Comment (PrithisIyyappan) [inline MAUI/Masked-Entry/MaskedEntry_Images/MaskedEntry_ReturnType.png:1]: change this image.

File: MAUI/Autocomplete/UI-Customization.md (hunk 2 above)
Comment (PrithisIyyappan) [reply MAUI/Autocomplete/UI-Customization.md:1242]: Add control name in text.

File: MAUI/Autocomplete/Images/UICustomization/ReturnType.png
Comment (PrithisIyyappan) [inline MAUI/Autocomplete/Images/UICustomization/ReturnType.png:1]: Change this image too.

File: MAUI/NumericEntry/Basic-Features.md [hunk 3]
```diff
@@ -268,4 +268,110 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
...
+sfNumericEntry.Value = 1000;
+sfNumericEntry.CustomFormat = "p";
+sfNumericEntry.PercentDisplayMode = PercentDisplayMode.Compute;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Default value of PercentDisplayMode is `Compute`.
+
+## Set the maximum number of decimal digits in SfNumericTextBox
```
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/Basic-Features.md:328]: 'SfNumericTextBox' change this.

File: MAUI/NumericEntry/GettingStarted_images/returntype.png
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/GettingStarted_images/returntype.png:1]: change this image too.

File: MAUI/NumericEntry/Basic-Features.md [hunk 4]
```diff
@@ -268,4 +268,110 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
 
 The following image illustrates the result of the above code:
 
-![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
\ No newline at end of file
+![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
+
+## PercentDisplayMode
```
Comment (PrithisIyyappan) [inline MAUI/NumericEntry/Basic-Features.md:273]: Modify the title.

File: MAUI/Autocomplete/UI-Customization.md (hunk 2 above)
Comment (SivaranjithN) [reply MAUI/Autocomplete/UI-Customization.md]: Addressed the comments.

File: MAUI/NumericEntry/Basic-Features.md (hunk 3 above)
Comment (SivaranjithN) [reply MAUI/NumericEntry/Basic-Features.md]: Addressed the comments.

File: MAUI/NumericEntry/Basic-Features.md [hunk 5]
```diff
@@ -268,4 +268,110 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
...
\ No newline at end of file
+![HorizontalTextAlignment](GettingStarted_images/textalignment.png)
+
+## Set the compute to percentage in SfNumericEntry
+
+When the [SfNumericEntry](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html) is in percentage format, the value can be displayed in two ways as follows:
+
+`Value`: Displays the actual value with percentage symbol.
+
+![.NET MAUI NumericEntry PercentDisplayMode](GettingStarted_images/percentdisplaymode_value.png)
```
Comment (Choza-rajan) [inline MAUI/NumericEntry/Basic-Features.md:279]: @SivaranjithN Add the output image after the code snippet

File: MAUI/NumericEntry/Basic-Features.md [hunk 6]
```diff
@@ -268,4 +268,110 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
...
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry sfNumericEntry = new SfNumericEntry();
+sfNumericEntry.ReturnType = ReturnType.Next;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI NumericEntry MaximumNumberDecimalDigits](GettingStarted_images/maximumnumberdecimaldigits.png)
```
[the remark this answers is not in the document]
Comment (SivaranjithN) [reply MAUI/NumericEntry/Basic-Features.md]: Changed the image.

File: MAUI/NumericEntry/Basic-Features.md (hunk 5 above)
Comment (SivaranjithN) [reply MAUI/NumericEntry/Basic-Features.md]: Addressed the comments.

File: MAUI/NumericEntry/Basic-Features.md [hunk 7]
```diff
@@ -268,4 +268,114 @@ sfNumericEntry.VerticalTextAlignment = TextAlignment.Start;
...
+
+![.NET MAUI NumericEntry PercentDisplayMode](GettingStarted_images/percentdisplaymode_compute.png)
+
+N> Default value of PercentDisplayMode is `Compute`.
+
+## Set the maximum number of decimal digits in SfNumericEntry
+
+The maximum number of digits to be displayed after the decimal point can be specified by using the `MaximumNumberDecimalDigits` property.
+
+N> The `MaximumNumberDecimalDigits` property can be provided with positive value only. Default value of MaximumNumberDecimalDigits is `2`.
```
Comment (Choza-rajan) [inline MAUI/NumericEntry/Basic-Features.md:332]: @SivaranjithN  Add the MaximumNumberDecimalDigits not working for custom format details also

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**